.github/workflows/test-gitpod.yml

```diff
@@ -0,0 +1,41 @@
+name: "Gitpod"
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - ".gitpod.yml"
+      - ".gitpod.Dockerfile"
+  pull_request:
+    branches: [main]
+    paths:
+      - ".gitpod.yml"
+      - ".gitpod.Dockerfile"
+
+env:
+  DAGGER_LOG_FORMAT: plain
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: "Expose GitHub Runtime"
+        uses: crazy-max/ghaction-github-runtime@v1
+
+      - name: "Set up dagger"
+        uses: dagger/dagger-for-github@v2
+        with:
+          install-only: true
+
+      - name: Build
+        env:
+          DAGGER_LOG_LEVEL: "debug"
+          DAGGER_LOG_FORMAT: "plain"
+          DAGGER_CACHE_TO: "type=gha,mode=max,scope=dagger-ci-build"
+          DAGGER_CACHE_FROM: "type=gha,scope=dagger-ci-build"
```

Comment:
Remove those one for now, we must use a hack to avoid overriding cache from other PR.
See #2120 for details